src/lang-pt_BR.js

```diff
@@ -0,0 +1,517 @@
+(function (janela) {
+  /**
+   * {virtualclass1} {virtualclass2} are elements you passes with getString function eg:-
+   *   virtualclass.lang.getString('operaBrowserIssue', ['opeara', 27]);
+   *   opera and 27 will be replaced over the {virtualclass1} and {virtualclass2} resepectively for particular line of language file.
+   * @type type
+   */
+  const message = {
+    ActiveAll: 'Ativar todos',
+    addcontext: 'Adicionar marcador de páginas',
+    Addoption: 'Adicionar opção',
+    addnew: 'Adicionar novo',
+    adminusr: 'Usuário admin',
+    appPrerequites: 'Validar requerimentos do sistema',
+    assign: 'Atribuir',
+    assignDisable: 'Retomar controles',
+    askplayMessage: '<span id="askplaymsg"> Devemos começar a sessão?</span><br /><span id="remaingmsg">Os dados restantes serão baixados em paralelo.</span>',
+    askplaymsg: '<span id="askplaymsg"> "Download em progresso, clique Play para iniciar </span>"',
+    audioInput: 'Entrada de áudio',
+    audiolatency: 'Latência de áudio',
+    audioOutput: 'Saída de áudio',
+    audioDisable: 'Desmutar',
+    audioEnable: 'Mutar',
+    addQuestion: 'Adicionar questão',
+    addContext: 'Adicionar Marcador de Páginas',
+    answer: 'Respostas',
+    action: 'Ações',
+    audioTest: 'Sua voz será gravada e tocada de volta para você. \n Pressione Ok e fale algo por uns segundos.',
+    Back: 'Voltar',
+    bandfast: 'Conectividade de rede ótima',
+    bandmedium: 'Conectividade de rede devagar',
+    bandslow: 'Falha na conectividade de rede',
+    bandwitdhImageNotFound: 'Download da imagem falhou, por favor clique em "Próximo" para continuar.',
+    becomeTeacher: 'Virar professor',
+    Binary_Limit_Exeed: 'Alerta: Alto uso BN',
+    bulkUserActions: 'Ações em massa do usuário',

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected linebreaks to be 'LF' but found 'CRLF'.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5482209)